Lesson1/Selim_Osmanov/HomeTask1.2/ht1_2lib.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <list>
+#include <cmath>
+void TakeNumbers(size_t Exponent, size_t Basis, std::list<size_t>& OutList){
```

Comment:
This function must return not the std::list, but some integer value.
std::list just should help you to solve the task.